.env

```diff
@@ -0,0 +1 @@
+VITE_API_READ_ACCESS_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJmZmQ5MTBhNDQxNzI5NmQ5MDczZjJlNmUzZjNlMDk1NyIsInN1YiI6IjY0YTJjZDhiMTEzODZjMDBhZGM3OWNjNSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.UCEqmnJ1SMmALiGWVThtFffX-_Kvj1qHkUlmjehNehk"
```

Comment:
add .env to gitignore
add .env.example instead
avoid sending your personal token to public repositories
VITE_API_READ_ACCESS_TOKEN = YOUR_PERSONAL_TOKEN

Reply:
please give a resource about using .env.example , I dont have any idea how can i add it and searching isnt clearly

Reply:
we fix this issue on meeting